runtime/src/bank.rs

```diff
@@ -3239,7 +3239,7 @@ impl Bank {
 
     fn recheck_compute_budget(self: &mut Bank) {
         let compute_budget = if OperatingMode::Stable == self.operating_mode() {
-            if self.epoch() >= u64::MAX - 1 {
+            if self.epoch() >= 91 {
```

Comment:
This appears to affect OperatingMode::Stable.  This should be OperatingMode::Preview for testnet

Reply:
yeah, I think so too.

Reply:
@ryo, looks like the same issue with `recheck_cross_program_support`

Reply:
aka, we just cold turned cpi on on testnet without gating?

Reply:
Well, when a cluster is running, I think gating is required to avoid different outcome depending updated/not-updated validators.

The exception is that the new different behavior is really rare to occur like some edge cases.

For clarity, it seems cpi is enabled on testnet without gating: https://github.com/solana-labs/solana/pull/11736/files#r478775897. Or, it did release along with some cluster whole restart? I think these large features should be introduced with gating.

I think gating for this feature is desired for testnet. In that way, we're also testing that the gating itself is working as expected. So, we can be rest assured for mainnet-beta.

Reply:
`Preview now introduces both loaders at epoch zero and also the new compute constraints